x-pack/plugins/data_visualizer/public/application/common/components/document_count_content/document_count_chart/document_count_chart.tsx

```diff
@@ -57,9 +57,12 @@ export const DocumentCountChart: FC<Props> = ({
   interval,
 }) => {
   const {
-    services: { data, uiSettings, fieldFormats },
+    services: { data, uiSettings, fieldFormats, charts },
   } = useDataVisualizerKibana();
 
+  const chartTheme = charts?.theme.useChartsTheme();
```

Comment:
Making `charts` non optional as per my previous comment means we can lose the `?` here

Reply:
Changed in https://github.com/elastic/kibana/pull/121286/commits/69c0bd38a8bbba9347ffc607f2fa6ad3fa55ac0d